bench.csv

```diff
@@ -2,22 +2,28 @@ name,cpu,mem,size
 Agora/Effects/Treasury Withdrawal Effect/effect/Simple,340268715,724428,3050
 Agora/Effects/Treasury Withdrawal Effect/effect/Simple with multiple treasuries ,570029812,1211300,3377
 Agora/Effects/Treasury Withdrawal Effect/effect/Mixed Assets,502351827,1071087,3242
-Agora/Effects/Governor Mutation Effect/validator/valid new governor datum/governor validator should pass,103830462,228928,7628
+Agora/Effects/Governor Mutation Effect/validator/valid new governor datum/governor validator should pass,103830462,228928,7629
 Agora/Effects/Governor Mutation Effect/validator/valid new governor datum/effect validator should pass,127968605,266935,3358
 Agora/Stake/policy/stakeCreation,59776675,126049,2116
 Agora/Stake/validator/stakeDepositWithdraw deposit,276249331,599197,4024
 Agora/Stake/validator/stakeDepositWithdraw withdraw,276249331,599197,4016
 Agora/Proposal/policy/proposalCreation,34784356,68894,1523
-Agora/Proposal/validator/cosignature/proposal,241204796,510319,4812
+Agora/Proposal/validator/cosignature/proposal,241651391,511819,5644
 Agora/Proposal/validator/cosignature/stake,186332635,402961,4561
-Agora/Proposal/validator/voting/proposal,239645722,489368,4820
+Agora/Proposal/validator/voting/proposal,240181636,491168,5652
 Agora/Proposal/validator/voting/stake,154223940,328703,4614
+Agora/Proposal/validator/advancing/successfully advance to next state/Draft -> VotringReady,131365724,260351,5030
+Agora/Proposal/validator/advancing/successfully advance to next state/VotingReady -> Locked,130643392,258848,5039
+Agora/Proposal/validator/advancing/successfully advance to next state/Locked -> Finished,132128827,262454,5039
+Agora/Proposal/validator/advancing/successfully advance to failed state: timeout/Draft -> Finished,129853757,257621,5032
+Agora/Proposal/validator/advancing/successfully advance to failed state: timeout/VotingReady -> Finished,128636280,254916,5039
+Agora/Proposal/validator/advancing/successfully advance to failed state: timeout/Locked -> Finished,129626570,257320,5039
 Agora/AuthorityToken/singleAuthorityTokenBurned/Correct simple,25177457,55883,806
 Agora/AuthorityToken/singleAuthorityTokenBurned/Correct many inputs,40266637,88241,900
 Agora/Treasury/Validator/Positive/Allows for effect changes,37343572,79744,1841
 Agora/AuthorityToken/singleAuthorityTokenBurned/Correct simple,25177457,55883,806
 Agora/AuthorityToken/singleAuthorityTokenBurned/Correct many inputs,40266637,88241,900
 Agora/Governor/policy/GST minting,57978053,120125,1833
-Agora/Governor/validator/proposal creation,330344593,681815,8143
-Agora/Governor/validator/GATs minting,431952116,934409,8266
-Agora/Governor/validator/mutate governor state,101019422,223202,7685
+Agora/Governor/validator/proposal creation,330344593,681815,8145
+Agora/Governor/validator/GATs minting,442720585,955552,8268
+Agora/Governor/validator/mutate governor state,101019422,223202,7686

```

Comment:
Script size is getting big :fearful:. But I have faith we can improve it again.

Reply:
Sure thing 😃